draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -320,18 +320,19 @@ verification key for the Issuer-signed JWT:
 
 - JWT Issuer Metadata: If the `iss` value contains an HTTPS URI, the recipient MUST
 obtain the public key using JWT Issuer Metadata as defined in (#jwt-issuer-metadata).
+- X.509 Certificates: The recipient MUST obtain the public key from the leaf X.509 certificate
+corresponding to the `x5c`, `x5c`, `x5t`, or `x5t#S256` JWT header parameters of the Issuer-signed JWT if one of them is present. In this case, the recipient MUST validate the X.509 certificate chain as follows:
+    - If the `iss` value contains a DNS name encoded as a URI using the DNS URI scheme [@RFC4501], the DNS name MUST match a `dNSName` Subject Alternative Name (SAN) [@RFC5280] entry of the leaf certificate.
+    - In all other cases, the `iss` value MUST match a `unifiedResourceIdentifier` SAN entry of the leaf certificate.
```

Comment:
AFAIK there's no unifiedResourceIdentifier in RFC5280 - it's uniformResourceIdentifier 

```suggestion
    - In all other cases, the `iss` value MUST match a `uniformResourceIdentifier` SAN entry of the leaf certificate.
```

Reply:
^ again